### ISSUE TITLE
Can't resolve './QueryMode.schema'

### ISSUE THREAD

**Issue by felixerdy** [omar-dulaimi/prisma-yup-generator#1]
### Bug description

Thank you for this great generator. Unfortunately, I'm getting the following error during an updateOne validation:
```
Module not found: Can't resolve './QueryMode.schema'
  2 | import * as Yup from 'yup'
  3 | import '../helpers/oneOfSchemas.helper.ts'
> 4 | import { QueryModeSchemaObject } from './QueryMode.schema'
  5 | import { NestedStringNullableFilterSchemaObject } from './NestedStringNullableFilter.schema'
  6 | 
  7 | export const StringNullableFilterSchemaObject = {

Import trace for requested module:
./prisma/generated/schemas/objects/AccountScalarWhereInput.schema.ts
./prisma/generated/schemas/objects/AccountUpdateManyWithoutUserInput.schema.ts
./prisma/generated/schemas/objects/UserUpdateWithoutQuestsInput.schema.ts
./prisma/generated/schemas/objects/UserUpdateOneRequiredWithoutQuestsInput.schema.ts
./prisma/generated/schemas/objects/QuestUpdateInput.schema.ts
./prisma/generated/schemas/updateOneQuest.schema.ts
./pages/api/quest/[id].ts
```

While taking a look into `StringNullableFilter.schema.ts` in `prisma/generated/schemas/objects` it seems like there is an incorrect import statement:
```
import { QueryModeSchemaObject } from './QueryMode.schema'
```

which should be:
```
import { QueryModeSchemaObject } from '../enums/QueryMode.schema'
```

### How to reproduce

running `await QuestUpdateOneSchema.validate({ ...body })`

### Prisma information

```
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model Account {
  id                String  @id @default(cuid())
  userId            String
  type              String
  provider          String
  providerAccountId String
  refresh_token     String? @db.Text
  access_token      String? @db.Text
  expires_at        Int?
  token_type        String?
  scope             String?
  id_token          String? @db.Text
  session_state     String?

  user User @relation(fields: [userId], references: [id], onDelete: Cascade)

  @@unique([provider, providerAccountId])
}

model Session {
  id           String   @id @default(cuid())
  sessionToken String   @unique
  userId       String
  expires      DateTime
  user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
}

model User {
  id            String    @id @default(cuid())
  name          String?
  email         String?   @unique
  emailVerified DateTime?
  image         String?
  accounts      Account[]
  sessions      Session[]
  Quests        Quest[]
}

model VerificationToken {
  identifier String
  token      String   @unique
  expires    DateTime

  @@unique([identifier, token])
}

model Quest {
  id          String  @id @default(uuid())
  name        String?
  description String?
  draft       Boolean @default(true)
  user        User    @relation(fields: [userId], references: [id])
  userId      String
}

generator yup {
  provider = "prisma-yup-generator"
}
```

### Environment & setup

- OS:  macOS 12.2.1
- Database: PostgreSQL 13
- Node.js version: v16.14.2 

### Prisma Version

```sh
➜ npx prisma version 
Environment variables loaded from .env
prisma                  : 3.13.0
@prisma/client          : 3.13.0
Current platform        : darwin-arm64
Query Engine (Node-API) : libquery-engine efdf9b1183dddfd4258cd181a72125755215ab7b (at node_modules/@prisma/engines/libquery_engine-darwin-arm64.dylib.node)
Migration Engine        : migration-engine-cli efdf9b1183dddfd4258cd181a72125755215ab7b (at node_modules/@prisma/engines/migration-engine-darwin-arm64)
Introspection Engine    : introspection-core efdf9b1183dddfd4258cd181a72125755215ab7b (at node_modules/@prisma/engines/introspection-engine-darwin-arm64)
Format Binary           : prisma-fmt efdf9b1183dddfd4258cd181a72125755215ab7b (at node_modules/@prisma/engines/prisma-fmt-darwin-arm64)
Default Engines Hash    : efdf9b1183dddfd4258cd181a72125755215ab7b
Studio                  : 0.459.0
```

**Comment by omar-dulaimi** [comment]
Hello  @felixerdy 
thank you for reporting this bug, I'll work on it as soon as possible. Possibly will be released today too.

Thank you for this library, I appreciate it!

**Comment by omar-dulaimi** [comment]
@felixerdy Should be fixed now with `prisma-yup-generator@0.0.0-rc.3`

Please feel free to submit any other issues you face :)
